libglusterfs/src/fd.c

```diff
@@ -462,7 +462,8 @@ fd_destroy(fd_t *fd, gf_boolean_t bound)
         for (i = 0; i < fd->xl_count; i++) {
             if (fd->_ctx[i].key) {
                 xl = fd->_ctx[i].xl_key;
-                old_THIS = THIS;
+                if (!old_THIS)
```

Comment:
nit: I do believe these lines (including line 467) should be under line 468 - only under that if condition.

Reply:
Done